spec/services/page_options_service_spec.rb

```diff
@@ -348,5 +348,36 @@
         end
       end
     end
+
+    context "with answer type of file" do
+      context "when file upload question does not contain guidance text" do
+        # this shouldn't be possible as the heading and text for guidance have to be set together
+        let(:page) { build :page, :with_file_upload_answer_type, :with_guidance }
+
+        it "returns the page heading" do
+          expect(page_options_service.all_options_for_answer_type).to include(
+            { key: { text: I18n.t("reports.form_or_questions_list_table.headings.question_text") },
+              value: { text: page.question_text } },
+          )
+        end
+      end
+
+      context "when file upload question contains guidance text" do
+        let(:page) { build :page, :with_guidance, :with_file_upload_answer_type }
```

Comment:
This maybe similar to @stephencdaly comments above.

I'm not sure how this context is different from the one on  line 353.

The page is the same but :with_guidance and :with_file_upload_answer_type reversed?

Reply:
I think this test is ok to be left as is once I've changed above to do what Stephen suggests